src/codec/audio/frame.rs

```diff
@@ -497,6 +499,36 @@ impl AudioFrame {
     pub(crate) fn as_ptr(&self) -> *const c_void {
         self.ptr
     }
+
+    /// Try to make this frame mutable. Returns AudioFrameMut if it can be made
+    /// into mutable without copying the data, otherwise returns AudioFrame.
+    pub fn try_into_mut(self) -> Result<AudioFrameMut, AudioFrame> {
+        let res = unsafe { ffw_frame_is_writable(self.ptr) };
+        return if res > 0 {
+            Ok(self.into_mut())
+        } else {
+            Err(self)
+        };
```

Comment:
Just to make Clippy happy :)

```suggestion
        if res > 0 {
            Ok(self.into_mut())
        } else {
            Err(self)
        }
```